src/publishDocumentationSnapshotOnlyIfNecessary.sh

```diff
@@ -26,6 +29,6 @@ else
   # update checksum file and trigger new documentation build and upload
   #
   echo "Creating and publishing documentation..."
-  cp --force "${current}" "${published}"
-  echo "./gradlew --scan gitPublishPush"
+  cp --force "${current}" "${published}"ca
```

Comment:
`ca`?

Reply:
Why so forcefully?

Reply:
`ca` ... _ceine ahnung_ ;-)

`--force` to overwrite existing one.